sql-statements/sql-statement-add-index.md

```diff
@@ -85,7 +85,10 @@ mysql> EXPLAIN SELECT * FROM t1 WHERE c1 = 3;
 
 ## MySQL compatibility
 
-* `FULLTEXT`, `HASH` and `SPATIAL` indexes are not supported.
+* TiDB accepts index types such as `HASH`, `BTREE` and `RTREE` in syntax but ignores them for compatibility reasons.
+* `SPATIAL` indexes are not supported.
+* `FULLTEXT` is only for syntax compatibility with MySQL and is not supported.
+* TiDB supports parsing the `FULLTEXT` syntax but does not support using the `FULLTEXT` indexes.
```

Comment:
I think lines 90 and 91 are saying the same thing now

Reply:
Ok deleted one